packages/react-components/src/WithWalletConnector.ts

```diff
@@ -301,30 +257,20 @@ export class WithWalletConnector extends Component<Props, State> implements Wall
         console.debug("WithWalletConnector: calling 'onDisconnected'", { connection, state: this.state });
         this.setState((state) => ({
             ...state,
-            activeConnection: connection === state.activeConnection ? undefined : state.activeConnection,
```

Comment:
After this change, we need a way to detect that a connection has disconnected in downstream state. One option is to have the key set of `connectedAccounts` consitute the set of live connections. This would require connections where there is no account to be stored with value `null`/`undefined`/empty string. A more easily understandable option is to expose the set explicitly.